pyproject.toml

```diff
@@ -59,7 +59,8 @@ dependencies = [
   "cachetools",
   "python-multipart",  # see https://www.starlette.io/#dependencies
   "arize-phoenix-evals>=0.13.1",
-  "pyyaml", # for OpenAPI
+  "fastapi",
+  "pydantic>=1.0,!=2.0.*,<3,", # exclude 2.0.* since it does not support the `json_encoders` configuration setting
```

Comment:
how much does this diverge from fastapi?

Reply:
The current version of `fastapi` officially has a lower bound of `1.7.4`, but it does not actually appear to be compatible with that version. See comment [here](https://github.com/Arize-ai/phoenix/issues/3855#issuecomment-2244036126).